packages/zowe-explorer-api/__tests__/__unit__/globals/Gui.unit.test.ts

```diff
@@ -153,23 +153,23 @@ describe("Gui unit tests", () => {
 
     it("can resolve a quick pick when accepted", async () => {
         // eslint-disable-next-line @typescript-eslint/no-unsafe-return
```

Comment:
should we be removing these that have been change to off or warn so when turned back on for fixing we are notified of location of items to be fixed?